tests/correct/incrementAndDecrement.rock

```diff
@@ -9,10 +11,12 @@ say my world
 knock my world down down down down down down
 say my world
 
+(This part tests that increment and decrement work on decimals)
```

Comment:
I would get rid of `This part tests that` - `(Increment and decrement work on decimals)` is descriptive enough